Configuration/NodeTypes.Overrides.yaml

```diff
@@ -0,0 +1,9 @@
+'Neos.Neos:Node':
+  options:
+    fusion:
+      prototypeGenerator: ~
```

Comment:
We have two options here: either disable the generators altogether or tune generators to follow the proposed best practices.
- Disabling is more beginner friendly, less magic to explain, people start to understand what really is going on. I'd vote for this option. :+1: 
- But for real world projects generators save you from typing, and that might be a very good practice to use them. Here are the generators that I came up with myself: https://github.com/Flowpack/fusion-bp/tree/master/Packages/Sites/Flowpack.FusionBP/Classes/Generators
Eventually it would be good to have some sort of official default generator that we ship by default, so it would be easier to understand other people's code. But yet again, generators magic is very hard to explain when teaching Neos.